core/src/main/java/org/openedx/core/presentation/catalog/WebViewLink.kt

```diff
@@ -0,0 +1,51 @@
+package org.openedx.core.presentation.catalog
+
+import android.net.Uri
+import org.openedx.core.extension.getQueryParams
+
+/**
+ * To parse and store links that we need within a WebView.
+ */
+class WebViewLink(
+    var authority: Authority,
+    var params: Map<String, String>
+) {
+    enum class Authority(val key: String) {
+        COURSE_INFO("course_info"),
+        PROGRAM_INFO("program_info"),
+        ENROLL("enroll"),
+        ENROLLED_PROGRAM_INFO("enrolled_program_info"),
+        ENROLLED_COURSE_INFO("enrolled_course_info"),
+        COURSE("course")
```

Comment:
please confirm, that these are useable in the future. if not better to remove.

Reply:
They will be used with the Programs WebView screen.